app/code/community/Oyst/OneClick/Block/OneClick.php

```diff
@@ -126,4 +126,55 @@ public function getProductType()
     {
         return Zend_Json::encode($this->escapeHtml($this->getProduct()->getTypeId()));
     }
+
+    /**
+     * Get custom wrapper class.
+     *
+     * @return null|string
+     */
+    public function getWrapperClass()
+    {
+        if ($class = Mage::getStoreConfig('oyst/oneclick/checkout_cart_button_class')) {
+            return 'class="' . $class . '"';
+        }
+
+        return null;
+    }
+
+    /**
+     * Add custom styles to button
+     *
+     * @param string $classPath
+     * @param string $stylesPath
+     *
+     * @return null|string
+     */
+    public function addButtonWrapperStyles($classPath, $stylesPath)
```

Comment:
remove this method, duplicate with https://github.com/oystparis/oyst-1click-magento/pull/252/files